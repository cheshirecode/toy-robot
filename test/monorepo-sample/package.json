{
  "private": true,
  "workspaces": [
    "packages/*"
  ],
  "dummyProps": {
    "dummyProps-key": "dummyProps-value"
  }
}
