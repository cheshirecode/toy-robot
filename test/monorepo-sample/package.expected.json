{
  "private": true,
  "dummyProps": {
    "dummyProps-key": "dummyProps-value"
  },
  "dependencies": {
    "webpack": "^4.8.1",
    "webpack-cli": "^3.0.8"
  },
  "devDependencies": {
    "babel-eslint": "^8.0.2",
    "eslint": "4.18.2",
    "eslint-config-prettier": "2.9.0",
    "eslint-config-recommended": "^3.0.0",
    "husky": "^0.14.3",
    "lint-staged": "^7.1.3",
    "prettier": "^1.13.4",
    "@rntw/eslint-config-lint": "0.1.0",
    "webpack-bundle-analyzer": "^2.13.1",
    "webpack-dev-server": "^3.1.4"
  }
}